src/path_info.cpp

```diff
@@ -158,6 +158,12 @@ void PATH_INFO::set_standard_filenames(void)
     update_pathname("obsolete-mods", FILENAMES["moddir"] + "obsolete-mods.json");
     update_pathname("defaultsounddir", FILENAMES["datadir"] + "sound");
 
+    // User's data directories
+    update_pathname("user_datadir", FILENAMES["user_dir"] + "data/");
+    update_pathname("user_gfxdir", FILENAMES["user_datadir"] + "gfx/");
```

Comment:
Aagh, html formatting strikes again.
What I meant to say is with the default install, user_dir is the build
directory, and gfx is in that directory, not under data/.  As a result, the
test for FILENAMES["user_datadir"] != FILENAMES["datadir"] is going to pass
when it shouldn't.